code/game/mecha/equipment/tools/tools.dm

```diff
@@ -726,6 +738,13 @@
 		send_byjax(chassis.occupant,"exosuit.browser","\ref[src]",src.get_equip_info())
 	return
 
+/obj/item/mecha_parts/mecha_equipment/gravcatapult/alt_action()
+	if(mode == 1)
```

Comment:
Are there defines for this ? If so, use them

Reply:
No defines; essentially the same thing is done in mecha_equipment/tools/red.